pynemo/nemo_bdy_extr_tm3.py

```diff
@@ -809,27 +896,38 @@ def time_interp(self, year, month):
         # Take the difference of the first two time enteries to get delta t
         
         del_t = time_counter[1] - time_counter[0]
-        dstep = 86400 / np.int(del_t)
+        dstep = 86400 // np.int(del_t)
       
         # TODO: put in a test to check all deltaT are the same otherwise throw 
         # an exception
     
         # If time freq. is greater than 86400s 
         # TODO put in an error handler for the unlikely event of frequency not a
         # multiple of 86400 | data are annual means
+        if self.key_vec == True:
+            if self.rot_dir == 'i':
+                varnams = [self.var_nam[0],]
+            else:
+                varnams = [self.var_nam[1],]
+        else:
+            varnams=self.var_nam
+
+       
```

Comment:
determine whether we're using source u and v to calc destination u or destination v